Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+<Project>
```

Comment:
I'm a bit concerned about creating a props at the root and also an artifacts directory in the same location.

Reply:
Artifacts output requires it to be at the solution root.

Reply:
To not have that file at the root you're fighting the tooling because Artifacts Output specifically wants that file at that location to enable it. If you define the property elsewhere it generates an error and tells you to move it.

I did try putting it in the existing files, and then the output was in the wrong place and needed defining in multiple places. I was fighting the tooling.

Having the file at the root fits with the .NET SDK defaults and what someone uninitiated walking up to the repo would expect to find.

Reply:
I would try a risk and battle test this solution.